lib/rust/flexer/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name    = "flexer"
 version = "0.1.0"
 authors = [
   "Enso Team <enso-dev@enso.org>",
-  "Ara Adkins <ara.adkins@enso.org"
+  "Ara Adkins <ara.adkins@enso.org>"
```

Comment:
Ara is part of Enso team, so there is no need to keep that separate. We are not doing it anywhere else. Lets reovethis entrty